### DISCUSSION TITLE
List installed terraform versions

### DISCUSSION THREAD

**Discussion by ginjo** [warrensbox/terraform-switcher#424] [Q&A] [answered]
Is there a tfswitch option to list installed terraforms?

**Comment by yermulnik** [comment, marked as answer]
Not that I know of.
What would be the use case for this feature?

**Reply by warrensbox** [reply]
If you have used `nvm`- it's similar to `nvm list` - https://github.com/nvm-sh/nvm?tab=readme-ov-file#listing-versions

It's nice to have, not necessary, nut nice to have. We shall see if others request this feature.

**Reply by yermulnik** [reply]
Ref: https://github.com/warrensbox/terraform-switcher/issues/270#issuecomment-2129788134

**Reply by warrensbox** [reply]
Ah I see. Thanks!
